xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -1709,17 +1709,18 @@ web.history.changes.restrictedInfo=For security reasons, at least one of the com
   restricted mode as it is not the current version. There may be differences and errors due to this.
 
 core.viewers.diff.title=Changes for page <a href="{1}">{0}</a>
+core.viewers.diff.change=From {0} to {1}
 core.viewers.diff.from=From version {0}
 core.viewers.diff.fromNew=From empty
 core.viewers.diff.to=To version {0}
 core.viewers.diff.editedBy=edited by {0}
 core.viewers.diff.editedOn=on {0}
 core.viewers.diff.editComment=Change comment:
 core.viewers.diff.noEditComment=There is no comment for this version
-core.viewers.diff.nextChange=Next change
-core.viewers.diff.previousChange=Previous change
-core.viewers.diff.nextVersion=Next version
-core.viewers.diff.previousVersion=Previous version
+core.viewers.diff.nextVersion.origin=Increase the origin version to {0}
+core.viewers.diff.previousVersion.origin=Decrease the origin version to {0}
+core.viewers.diff.nextVersion.target=Increase the target version to {0}
+core.viewers.diff.previousVersion.target=Decrease the target version to {0}
```

Comment:
I'm not super happy with the terms "origin version" and "target version". What do you think about "base version" and "comparison version"? Alternatively, we could also keep "target" and just change "origin" to "base". Any opinions?

Reply:
I personally would keep it like that for now. Maybe it's just me but I do find it more clear than "base version" and "comparison version" to know which is which without viewing the UI.

Reply:
I agree that 'origin' was an awkward way to put a name on this concept.

I think the 'base' version could be thought of as being the 1.0 . I wanted to get names that conveyed their role in calculating the diff. 

Thinking about it now, it seems that 'source' is better than 'origin':
 source version -> target version

Do you think it's better than 'base'?
__
I don't like too much 'comparison version', because the comparison word does not give an idea of the role in this comparison: base (base of comparison), origin (origin of the comparison), and target (target for the comparison) all bring more information that is useful.

In https://github.com/xwiki/xwiki-platform/pull/2323/commits/ba9d2007631b73a1d961ccd0b06042fa3427a6ae I updated ~~origin~~ --> base